### PR TITLE
Updated Uransus's moons

### LINKED ISSUE #1: Uranus's moons
The information about Uranus's moon is incomplete!

### PR DESCRIPTION
**Pull request by dgasmith**
Added all 27 of the moons, closes #1.

### CODE REVIEW

File: Uranus [hunk 1]
```diff
@@ -1 +1,5 @@
-Uranus 27 moons: Titania Oberon Umbriel Ariel Miranda
+Uranus 27 moons:
+Cordelia, Ophelia, Bianca, Cressida, Desdemona, Juliet, Portia, Rosalind,
+Cupid, Belinda, Perdita, Puck, Mab, Miranda, Ariel, Umbriel, Titania and
+Oberon, Francisco, Caliban, Stephano, Trinculo, Sycorax, Margaret, Prospero,
+Setebos and Ferdinand.
```
Comment (jchodera) [inline Uranus]: Could you add the Oxford (serial) comma?

Comment (dgasmith) [reply Uranus]: But wikipedia didn't do it.

Comment (jchodera) [reply Uranus]: We want to avoid ambiguity whenever possible:
https://en.wikipedia.org/wiki/Serial_comma#Unresolved_ambiguity

**Review by jchodera — APPROVED** [review]
LGTM

**Comment by jchodera** [comment]
Thanks again for fixing this!

**Pull request merged**